bot/src/stake_pool_v0.rs

```diff
@@ -24,6 +25,9 @@ const MIN_STAKE_DELEGATION: u64 = 1000000000;
 // Delegation rent amount. Need
 const DELEGATION_RENT: u64 = 2282880;
 
+// Do not move more than this amount of stake per epoch
+const MAX_SOL_MOVEMENT_PER_EPOCH: u64 = 5_000_000;
```

Comment:
wdyt of making this a percentage and resolving the actual number of tokens at runtime?  this value isn't going to work for all implementations.

NOTE: _highly_ recommend staying in integers. that is, `MAX_SOL_PCT_INT * total_staked / 100` as opposed to `MAX_SOL_PCT_FLT * total_staked`

Reply:
> wdyt of making this a percentage and resolving the actual number of tokens at runtime? this value isn't going to work for all implementations.

Yeah, this is at best a stopgap. Wasn't sure offhand how to get the actual number of tokens; could you point me in the right direction?